features/bootstrap/FeatureContext.php

```diff
@@ -94,6 +94,8 @@ public function createPayloadOptions(TableNode $tableNode): void
             isset($data['forceAccount']) ? (bool)$data['forceAccount'] : null,
             $data['returnUrl'] ? new ReturnUrl(null, $data['returnUrl']) : null,
             isset($data['pathfinding']) ? (bool)$data['pathfinding'] : null,
+            isset($data['pathfindingFallback']) ? (bool)$data['pathfindingFallback'] : null,
+            isset($data['forceNetwork']) ? $data['forceNetwork'] : null,
```

Comment:
You can null-coalesce here, as in:
```
$data['forceNetwork'] ?? null
```

Reply:
Applied the null-coalescing to `forceNetwork`.